packages/cubejs-backend-cloud/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": [
+    "src",
+    "test"
+  ],
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": ".",
+    "baseUrl": ".",
+    "allowJs": true,
+    "checkJs": true,
+    "noImplicitAny": false,
```

Comment:
I think it's better to enable it. Don't forget that it's possible to do `Record<string, any>`.